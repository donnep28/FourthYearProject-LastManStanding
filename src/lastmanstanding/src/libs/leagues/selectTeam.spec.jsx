import React from 'react';
import {shallow} from 'enzyme';
import SelectTeam from './selectTeam';

describe('pickTeam', () => {
  beforeAll(() => {
    jest.resetAllMocks();
  });

  const props = {
    submitToggle: true,
    selectedTeam: 'Manchest United FC',
    sub: 'mockSub',
    leagueID: 'mockLeagueId',
    setPermPick: jest.fn(),
    submitTeam: jest.fn(),
  };
  describe('Snapshot', () => {
    it('renders teams', () => {
      const component = shallow(
          <SelectTeam {...props}/>,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
