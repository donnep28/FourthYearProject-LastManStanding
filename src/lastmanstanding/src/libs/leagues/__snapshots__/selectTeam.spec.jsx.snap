// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pickTeam Snapshot renders teams 1`] = `
<WithStyles(ForwardRef(Button))
  data-automation="pickTeam_submitButton"
  onClick={[Function]}
>
  Select Team
</WithStyles(ForwardRef(Button))>
`;
